### PR TITLE
simple segmentation example

### PR DESCRIPTION
**Pull request by simonkassel**
This PR adds an example of a simple building segmentation experiment run on the Spacenet Vegas dataset.

### CODE REVIEW

File: spacenet/simple_segmentation.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+class SpacenetVegasSimpleSegmentation(rv.ExperimentSet):
+
+    def exp_main(self, root_uri, test='False'):
+        """Run an experiment on the Spacenet Vegas building dataset.
+
+        This is a simple example of how to do semantic segmentation.
+
+        Args:
+            root_uri: (str): root of where to put output
```
Comment (lewfish) [inline spacenet/simple_segmentation.py]: Should a add PyDoc for `test` arg

File: spacenet/simple_segmentation.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                                    .with_task(task) \
+                                    .with_model_defaults(rv.MOBILENET_V2) \
+                                    .with_num_steps(num_steps) \
+                                    .with_batch_size(batch_size) \
+                                    .with_debug(False) \
+                                    .build()
+
+        def build_scene(id):
+            train_image_uri = os.path.join(base_uri, raster_dir,
+
```
Comment (lewfish) [inline spacenet/simple_segmentation.py]: Extra newline here

File: README.md [hunk 3]
```diff
@@ -221,6 +221,65 @@ Viewing the validation scene results for scene ID `013022232023` looks like this
 
 ![QGIS results explorer](img/qgis-spacenet-cc.png)
 
+## Spacenet Vegas Simple Semantic Segmentation
```
Comment (lewfish) [inline README.md]: Should add an entry for this in the table of content at the top of the page.

File: spacenet/simple_segmentation.py (hunk 1 above)
Comment (simonkassel) [reply spacenet/simple_segmentation.py]: done

**Comment by simonkassel** [comment]
ok, I made these changes, should I merge this?

**Pull request merged**